sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -499,14 +499,16 @@ object SQLConf {
       .intConf
       .createWithDefault(40)
 
-  val VECTORIZED_AGG_MAP_MAX_COLUMNS =
-    SQLConfigBuilder("spark.sql.codegen.aggregate.map.columns.max")
```

Comment:
Let's also make sure that all references to the old config are also appropriately modified.